src/app/utils/badge-color.ts

```diff
@@ -0,0 +1,26 @@
+import { getTheme } from 'office-ui-fabric-react';
+
+export function getMethodStyle (method: string) {
```

Comment:
This function will read better if renamed to `getStyleFor(method: string) {...}`